src/content/introduction.md

```diff
@@ -33,7 +33,19 @@
 ---
 
 ## In this talk
-- Setting up forms to work with Redux
+- Your forms + Redux magic
 - Adding form data to state
 - Supporting multi-entry fields
 - Framework-agnostic validation with selectors
+
+---
+
+## Not in this talk
+- Redux 101
+
```

Comment:
I feel like we mention most of these things in the project dependencies but we also explain them as much as they're needed... Redux is the only thing we don't really give an overview of. Then again, maybe we should give an overview?